x-pack/plugins/security_solution/cypress/e2e/network/inspect.cy.ts

```diff
@@ -17,10 +17,11 @@ describe('Inspect', () => {
   context('Network stats and tables', () => {
     before(() => {
       cleanKibana();
-      loginAndWaitForPage(NETWORK_URL);
+      cy.task('esArchiverLoad', 'network');
```

Comment:
Is this needed?

Reply:
yes, after we have introduced `parallel.ts`, which setup a new stack instance per spec file we need to load data required for the test, I have aligned it with `main` branch